service.yml

```diff
@@ -0,0 +1,13 @@
+name: rest-utils
+lang: unknown
```

Comment:
@cprovencher should language be java with specific version

Reply:
Already answered here https://github.com/confluentinc/ce-kafka-rest/pull/548#discussion_r1387151749